fossology/fossology
# feat(licenseRef): add or update license texts from SPDX

## Description

Update license texts from SPDX.

### Changes

update licenseref  file.

## How to test

* Install FOSSology.
* Check if new licenses exists in database.

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2175"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
